src/TsvHttp.h

```diff
@@ -8,6 +8,7 @@
 #include "logging/WriterBackend.h"
 #include "threading/formatters/Ascii.h"
 #include "zlib.h"
+#include "Desc.h"
```

Comment:
I had to add this. The first time I tried to compile after making all the other changes, it failed complaining about not having the definition of `ODesc`. Rather than try to hunt down why it worked without this in older Zeek versions I instead just hunted down that this was where it's defined this `#include` shut it up.